features/upgrade/storage/upgrade.feature

```diff
@@ -179,13 +179,19 @@ Feature: Storage upgrade tests
   @upgrade-prepare
   @users=upuser1,upuser2
   @admin
+  @destructive
   Scenario: Snapshot operator should be in available status after upgrade and can created pod with snapshot - prepare
     Given the master version >= "4.4"
  
     #Deploy csi hostpath driver 
-    When I run the :new_project client command with:
-      | project_name | csihostpath |
-    
+    When I run the :oadm_new_project admin command with:
+      | project_name  | csihostpath            |
+      | node_selector | app=csi-hostpathplugin |
+      | admin         | <%= user.name %>       |
+ 
+    Given I store the ready and schedulable nodes in the :nodes clipboard
+    And label "app=csi-hostpathplugin" is added to the "<%= cb.nodes[0].name %>" node
```

Comment:
1. Looks there is no step to save some nodes in the `cb`
2. I don't think we need to use a different label, just create the `csihostpath` project with the node label, so we don't need to mark the test case is a destructive case?